ChangeLog.d/driver-only-hashes.txt

```diff
@@ -8,12 +8,16 @@ Features
      are only provided by PSA drivers. In these configurations, you need to
      call `psa_crypto_init()` before you call any function from those
      modules; this is not required in configurations where the built-in
-     implementation is still available. Note that some crypto modules and
-     features still depend on the built-in implementation of hashes:
-     MBEDTLS_HKDF_C (but the PSA HKDF function do not depend on it),
-     MBEDTLS_ENTROPY_C, MBEDTLS_HMAC_DRBG_C and MBEDTLS_ECDSA_DETERMINISTIC.
-     In particular, for now, compiling without built-in hashes requires use
-     of MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG.
+     implementation is still available. Note that for modules that use MD
+     (RSA, PKCS5, PKCS12, EC J-PAKE) in builds that have MBEDTLS_MD_C enabled,
+     all hashes used with those modules need to be built-in, as drivers are only
```

Comment:
Another example of a long sentence that's hard to follow: “modules that use MD (…) in builds that have MBEDTLS_MD_C enabled” — the context and the beginning makes the reader think this is about the case with MBEDTLS_MD_C disabled, but it's about the case where it is enabled. It would be easier to follow if the sentence started with “When MBEDTLS_MD_C is enabled”.

Reply:
In line with what I said above, a lot of sentences would automatically get a lot simpler if we moved the info to `mbedtls_config.h` and described only one module at a time.